app/controllers/receive_message_controller.rb

```diff
@@ -1,7 +1,10 @@
 class ReceiveMessageController < ApplicationController
   def receive
     from_number = params['From']
-    message     = User.new_conversation(from_number).next
+    user        = User.new(number: from_number)
+    message     = user.next_message
+
+    user.increment_conversation
```

Comment:
Yes, this is where the next change would be, and we would need a flag on the user or something to know if a reminder was sent and depending on that reply with the next or the same message